packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -35,7 +35,10 @@ const SearchSuggestions = ({
 
   if (preQueryHits) {
     preQueryHits.forEach((hit) => {
-      preQuerySuggestions.push(hit);
+      // or filter out here?
+      if (hit.content_type !== 'video') {
+        preQuerySuggestions.push(hit);
+      }
```

Comment:
I think you're on the right path with modifying the filters passed to the `SearchBox` component, so that it is passed along to both [here](https://github.com/openedx/frontend-enterprise/blob/21ce6f6fc1a6e9abd3510d64226c4e8c2921aea7/packages/catalog-search/src/SearchBox.jsx#L111) and [here](https://github.com/openedx/frontend-enterprise/blob/21ce6f6fc1a6e9abd3510d64226c4e8c2921aea7/packages/catalog-search/src/SearchBox.jsx#L126).

tl;dr; add the addtl filters as far upstream as possible so that `hits` never include unsupported content types to begin with :)